ui/src/App/index.jsx

```diff
@@ -89,6 +89,13 @@ export default class App extends Component {
     options: {
       reconnect: true,
       lazy: true,
+      connectionParams: async () => {
+        const user = await this.authController.getUser();
+
+        return {
+          Authorization: `Bearer ${btoa(JSON.stringify(user.credentials))}`,
```

Comment:
can you please check if this is empty or contains something when user is not logged in?

Reply:
Resolved: Added a null check for this.